test/performance/vectorize.cpp

```diff
@@ -95,15 +97,14 @@ int main(int argc, char **argv) {
     bool ok = true;
 
     // Only native vector widths for now
-    ok = ok && test<float>(4);
```

Comment:
Note that the original code effectively tested both sse and avx performance avx machines. Not sure if that really matters.